tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/LogsInjectionTestBase.cs

```diff
@@ -149,7 +172,16 @@ public void ValidateLogCorrelation(
                 var versionRegex = string.Format(test.RegexFormat, versionProperty, @"""1.0.0""");
                 var envRegex = string.Format(test.RegexFormat, envProperty, @"""integration_tests""");
                 var serviceRegex = string.Format(test.RegexFormat, serviceProperty, @$"""{EnvironmentHelper.FullSampleName}""");
-                var traceIdRegex = string.Format(test.RegexFormat, traceIdProperty, @"("")?(\d\d+)(?(1)\1|)"); // Match a string of digits or string of digits surrounded by double quotes. See https://stackoverflow.com/a/3569031
+                string traceIdRegex;
+                if (use128Bits)
+                {
+                    traceIdRegex = string.Format(test.RegexFormat, traceIdProperty, @"\""([0-9a-f]{32})\"""); // Match a string of digits or hex character - must be 32. Group 1 has ID
```

Comment:
Does this one also need the backreferencing like the ones below? "string of digits or string of digits surrounded by double quotes"

Reply:
Yep added!